src/styles/embeds/sass/components/select.css

```diff
@@ -24,7 +24,7 @@
 
 .shopify-buy__option-select {
   & + & {
-    margin-top: $gutter / 2;
+    margin-top: var(--gutter) / 2;
```

Comment:
Interesting side affect of the nesting.
this code is generating the following, which included extra empty block.

```css
.shopify-buy__option-select { } 
.shopify-buy__option-select + .shopify-buy__option-select {   margin-top: 15px / 2; }
```